element/index.js

```diff
@@ -65,3 +66,35 @@ export function renderToString( element ) {
 
 	return renderToStaticMarkup( element );
 }
+
+/**
+ * Concat two React children objects
+ *
+ * @param  {?Object} children1 First children value
+ * @param  {?Object} children2 Second children value
+ *
+ * @return {Array}             The concatenation
+ */
+export function concatChildren( children1, children2 ) {
```

Comment:
> I think it'd be easy enough and a nice feature to support a spread value of ...childrens, so two or more set of children can be concatenated

Great idea!

---

I've tried your version but it's not working. I'm finding the `toArray` to be necessary. I'm unsure if it may be a problem coming from another place (parser or anything), but you can easily reproduce a bug by backspacing from the start of the second test block "I imagine..."